.gitignore

```diff
@@ -54,12 +54,3 @@ coverage
 Network Trash Folder
 Temporary Items
 .apdisk
-# Devenv
-.devenv*
-devenv.local.nix
-
-# direnv
-.direnv
-
-# pre-commit
-.pre-commit-config.yaml
```

Comment:
@hakenprog I'm removing the devenv setup because it is unstable due to Playwright specs, so it is not good to recommend this setup for now in the README. I would open a future PR adding support again when possible. For now, it is okay to keep using asdf for this project.